### PR TITLE
Clean up errors, tokens

### PR DESCRIPTION
**Pull request by dekz**
### Fix cyclic import

There was a cyclic import dependency between `constants` and `config`. This resulted in unexpected `undefined` when `constants` attempted to read `config` (`CHAIN_ID` was undefined). Resulting in Bridges being disabled.

### If the request is an errror (>=400), log the `errorBody`
Previously 
```json
{
  "level": "info",
  "time": 1578871838735,
  "pid": 44899,
  "hostname": "Macintosh-2.local",
  "req": {
    "url": "/swap/quote",
    "method": "GET",
...
    },
    "body": {},
    "params": {},
    "query": {
      "buyToken": "WETH",
      "sellToken": "DAI",
      "buyAmount": "1000000000000000000000"
    }
  },
  "res": {
    "statusCode": 400,
    "statusMessage": "Bad Request"
  },
  "responseTime": 4381,
  "timestamp": 1578871838735,
  "v": 1
}

```

Now
```json
{
  "level": "info",
  "time": 1578873524070,
  "pid": 61012,
  "hostname": "Macintosh-2.local",
  "req": {
    "url": "/swap/v0/quote",
    "method": "GET",
...
    },
    "body": {},
    "params": {},
    "query": {
      "buyToken": "WETH",
      "sellToken": "DAI",
      "buyAmount": "1000000000000000000000"
    }
  },
  "res": {
    "statusCode": 400,
    "statusMessage": "Bad Request",
    "errorBody": "{\"code\":100,\"reason\":\"Validation Failed\",\"validationErrors\":[{\"field\":\"buyAmount\",\"code\":1004,\"reason\":\"INSUFFICIENT_ASSET_LIQUIDITY\"}]}"
  },
  "responseTime": 1881,
  "timestamp": 1578873524070,
  "v": 1
}

```

If it's successful we do not log the body:
```json
{"level":"info","time":1578874153331,"pid":69328,"hostname":"Macintosh-2.local","req":{"url":"/swap/v0/quote","method":"GET","headers":{"user-agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.88 Safari/537.36","host":"localhost:3000"},"body":{},"params":{},"query":{"buyToken":"weth","sellToken":"DAI","buyAmount":"1000000000000000"}},"res":{"statusCode":200,"statusMessage":"OK"},"responseTime":3292,"timestamp":1578874153331,"v":1}
```

### Map `NO_OPTIMAL_PATH` error to `InsufficientLiquidityError`, return 4xx:
```
curl 'http://localhost:3000/swap/v0/quote?buyToken=WETH&sellToken=DAI&buyAmount=1000000000000000000000'

{"code":100,"reason":"Validation Failed","validationErrors":[{"field":"buyAmount","code":1004,"reason":"INSUFFICIENT_ASSET_LIQUIDITY"}]}
```

### Throw appropriate API Error when token is not found.
```
curl 'http://localhost:3000/swap/v0/quote?buyToken=BAD&sellToken=DAI&buyAmount=1000000000000000000000'

{"code":100,"reason":"Validation Failed","validationErrors":[{"field":"buyToken","code":1004,"reason":"Could not find token BAD"}]}
```

### Handle mixed/lowercase symbols.
```
curl 'http://localhost:3000/swap/v0/quote?buyToken=weth&sellToken=Dai&buyAmount=1000000000000000'

{"price":"0.006775128311947509","to":"0x61935cbdd02287b511119ddb11aeb42f1593b7ef"...
```

### Tokens update
Update Synth/MLN addresses. Added USDT, removed ICN/ICX/AION which do not have liquidity.

### CODE REVIEW

File: src/handlers/swap_handlers.ts [hunk 1]
```diff
@@ -56,12 +57,15 @@ export class SwapHandlers {
...
+            ) {
                 throw new ValidationError([
                     {
                         field: buyAmount ? 'buyAmount' : 'sellAmount',
                         code: ValidationErrorCodes.ValueOutOfRange,
-                        reason: e.message,
+                        reason: SwapQuoterError.InsufficientAssetLiquidity,
                     },
                 ]);
             }
```
Comment (fragosti) [inline src/handlers/swap_handlers.ts]: Should we distinguish between the two cases for our own purpose?

File: src/middleware/request_logger.ts [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
  * log middleware
  */
 export function requestLogger(): core.RequestHandler {
     const handler = (req: express.Request, res: express.Response, next: core.NextFunction) => {
+        const origSend = res.send;
+        let cachedBody: any;
+        res.send = (body?: any): express.Response => {
+            cachedBody = body;
+            return origSend.bind(res)(body);
+        };
```
Comment (fragosti) [inline src/middleware/request_logger.ts]: Is this necessary? Should we be resetting `res.send` afterwards?

File: src/utils/token_metadata_utils.ts [hunk 3]
```diff
@@ -15,8 +15,8 @@ export function getTokenMetadataIfExists(tokenAddressOrSymbol: string, chainId:
             tm => tm.tokenAddresses[chainId].toLowerCase() === tokenAddressOrSymbol.toLowerCase(),
         );
     } else {
-        const normalizedSymbol = tokenAddressOrSymbol === ETH_SYMBOL ? 'WETH' : tokenAddressOrSymbol;
-        entry = TokenMetadatasForChains.find(tm => tm.symbol === normalizedSymbol);
+        const normalizedSymbol = (isETHSymbol(tokenAddressOrSymbol) ? 'WETH' : tokenAddressOrSymbol).toLowerCase();
+        entry = TokenMetadatasForChains.find(tm => tm.symbol.toLowerCase() === normalizedSymbol);
```
Comment (fragosti) [inline src/utils/token_metadata_utils.ts]: Nice

File: src/middleware/request_logger.ts (hunk 2 above)
Comment (dekz) [reply src/middleware/request_logger.ts]: Yeah since we need to hook into it, then call the original.

**Pull request merged**